docs/network-iot/run-an-lns/configure-chirpstack.mdx

```diff
@@ -45,6 +45,69 @@ Modify the following sections:
 - Under `[gateway]`:
   - Add `allow_unknown_gateways=true`
 
+
+### Example chirpstack.toml with helium additions.
```

Comment:
Let's move this to the bottom of the page since it encompasses the other topics that come after this section (Adding multiple regions, etc)